app/pb_kits/playbook/pb_card/_card_test.jsx

```diff
@@ -0,0 +1,105 @@
+/* @flow */
+
+import React from 'react'
+import classnames from 'classnames'
+import { buildCss } from '../utilities/props'
+
+//  Card props
+/*
+type CardPropTypes = {
+  children: Array<React.ReactNode> | React.ReactNode,
+  className?: String,
+  header?: {
+    padding?: 'none' | 'xs' | 'sm' | 'md' | 'lg' | 'xl',
+    color?: String
+  }
+  bodyPadding?: 'none' | 'xs' | 'sm' | 'md' | 'lg' | 'xl',
+}
+
+// Establishing css classes
+const CardTest = ({
+  children,
+  className,
+  header = {},
+  bodyPadding = 'md',
+}: CardPropTypes) => {
+	const mainCSS = buildCss('pb_card_kit', className)
+  const headerCSS = buildCss('pb_card_header_kit', {
+		[`header_${header.padding}`]: header.padding,
+		[`header_${header.color}`]: header.color,
+	})
+  const bodyCSS = buildCss('pb_card_body_kit', bodyPadding)
+}
+*/
+
+// FindByType util
+const findByType = (children, component) => {
+const result = [];
+/* This is the array of result since Article can have multiple times the same sub-component */
+const type = [component.displayName] || [component.name];
+/* We can store the actual name of the component through the displayName or name property of our sub-component */
+React.Children.forEach(children, child => {
+    const childType =
+    child && child.type && (child.type.displayName || child.type.name);
+    if (type.includes(childType)) {
+    result.push(child);
+    }
+});
+
+/* Then we go through each React children, if one of matches the name of the sub-component we’re looking for we put it in the result array */
+return result[0];
+};
```

Comment:
This looks great! If I'm understanding correctly, this is being used as a boolean that will resolve to `true` if a header should be displayed, and `false` if it should not be displayed. If so, the `.find` method could also work! It would return `undefined` if the `childType` was not there, then return the first object it finds if any match the criteria. Something like this: 

`children.find(({ type }) => (type.displayName || type.name) === childType)`

Reply:
Also - I think it may be possible that this function exists already without having to declare it...unclear though